### PR TITLE
Add Event indexer

### PR DESCRIPTION
**Pull request by michalsidzej**
Resolves L2B-2863, L2B-2874

### CODE REVIEW

File: packages/backend/src/indexers/EventIndexer.test.ts [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+      expect(blockchainClient.getBlock).toHaveBeenCalledTimes(1)
+      expect(blockchainClient.getBlock).toHaveBeenCalledWith(
+        startBlock + maxEventRange,
+      )
+      expect(blockchainClient.getLogsBatch).toHaveBeenCalledTimes(1)
+      expect(blockchainClient.getLogsBatch).toHaveBeenCalledWith(
+        address,
+        topics,
+        startBlock,
+        startBlock + 10,
```
Comment (sdlyy) [inline packages/backend/src/indexers/EventIndexer.test.ts:71]: Let's limit magic numbers
```suggestion
        startBlock + maxEventRange,
```

File: packages/backend/src/indexers/EventIndexer.ts
Comment (sdlyy) [inline packages/backend/src/indexers/EventIndexer.ts:1]: Let's sync regarding event indexer tmrw

**Review by sdlyy — CHANGES_REQUESTED** [review]
LG, few small things - let's discuss event indexer implementation since I want to be on the same page regarding final data flow

File: packages/backend/src/peripherals/database/EventRepository.test.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      {
+        blockNumber: 4,
+        chainId,
+      },
+      {
+        blockNumber: 6,
+        chainId,
+      },
+    ])
+  })
```
[the remark this answers is not in the document]
Comment (michalsidzej) [reply packages/backend/src/peripherals/database/EventRepository.test.ts:42]: Good point

File: packages/backend/src/indexers/EventIndexer.test.ts (hunk 1 above)
Comment (michalsidzej) [reply packages/backend/src/indexers/EventIndexer.test.ts]: [f749fae](https://github.com/l2beat/lz-monitoring/pull/45/commits/f749fae1338885babc9e10cf309ea107fb655626)

File: packages/backend/src/peripherals/database/EventRepository.test.ts
Comment (sdlyy) [inline packages/backend/src/peripherals/database/EventRepository.test.ts:1]: Let's seal `deleteAfter`

Comment (michalsidzej) [reply packages/backend/src/peripherals/database/EventRepository.test.ts:1]: [4cbc7fa](https://github.com/l2beat/lz-monitoring/pull/45/commits/4cbc7fac58cecd1bdafba26aa1ab5a50bfb02f65)

**Review by sdlyy — APPROVED** [review]
n i c e 🚀

**Pull request merged**